bigdata-core/bigdata-sails/src/java/com/bigdata/rdf/sail/BigdataSail.java

```diff
@@ -5426,15 +5424,14 @@ public synchronized void close() throws SailException {
 
     } // class BigdataSailReadOnlyConnection
 
-//FIXME:  Sesame 2.8 not used for 2.1.4 Release
-//	@Override
-//	public List<IsolationLevel> getSupportedIsolationLevels() {
-//		return Arrays.<IsolationLevel>asList(IsolationLevels.READ_UNCOMMITTED, IsolationLevels.SNAPSHOT_READ);
-//	}
+	@Override
+	public List<IsolationLevel> getSupportedIsolationLevels() {
+		return Arrays.<IsolationLevel>asList(IsolationLevels.READ_UNCOMMITTED, IsolationLevels.SNAPSHOT_READ);
```

Comment:
Only snapshot

Reply:
Is it READ_COMMITTED or SNAPSHOT_READ? Because Blazegraph has READ_COMMITTED but I am not sure it's the same meaning.